runtime/ztests/op/summarize/collect-union.yaml

```
@@ -0,0 +1,11 @@
+zed: summarize collect(this) | sort this
```

`sort` does nothing here since the output is a single value.
```suggestion
zed: summarize collect(this)
```